tools/circleci/docker_build_img/build.sh

```diff
@@ -12,37 +13,24 @@ get_pip_reqs() {
 
 docker_build() {
     local pip_reqs="$1"
-    local py_version=$2
-    local image=$ECR_IMAGE_URL:$py_version
+    local image=$ECR_IMAGE_URL:$PY_VERSION
     echo "Building docker image: $image"
     docker build -t $image \
         --build-arg PIP_REQS="$pip_reqs" \
-        --build-arg PY_VERSION=$py_version \
+        --build-arg PY_VERSION=$PY_VERSION \
         .
 }
 
 docker_push() {
-    local py_version=$1
-    local image=$ECR_IMAGE_URL:$py_version
+    local image=$ECR_IMAGE_URL:$PY_VERSION
     echo "Pushing docker image: $image"
     docker push $image
 }
 
 main() {
     local pip_reqs="$(get_pip_reqs $KEDRO_REPO)"
-
-    # Image for python 3.6
-    docker_build "$pip_reqs" 3.6
-    docker_push 3.6
-
-    # Image for python 3.7
-    docker_build "$pip_reqs" 3.7
-    docker_push 3.7
-
-    # Image for python 3.8
-    docker_build "$pip_reqs" 3.8
-    docker_push 3.8
-
+    docker_build "$pip_reqs"
```

Comment:
I've altered this script so it runs for just one Python version at a time. This matches how the rest of the CircleCI config works and means we can have exactly the same script on main and develop.